Automattic/wp-calypso
# Tests- Add preview theme tests and rename existing tests

#### Changes proposed in this Pull Request

* This change adds the visual regression tests for the theme demos. 

It works be getting the list of themes from the api, saving them to a file, and then adding the test scenarios for each one. 

* I also changed the name of the tests so that it was more clear what they were doing. This caused me to have to redo all of the reference images since the names changed.

#### Testing instructions



* Make sure that the tests pass in [TeamCity](https://teamcity.a8c.com/viewType.html?buildTypeId=calypso_VisualRegressionTests&branch_calypso_WPComTests=add%2Fmore-vr-tess&tab=buildTypeStatusDiv) 



Related to #
